pyvrp/cpp/Individual.cpp

```diff
@@ -1,5 +1,6 @@
 #include "Individual.h"
 #include "ProblemData.h"
+#include "TimeWindowSegment.h"
```

Comment:
I think this is not a bad thing. In my opinion, we want to avoid coupling primarily to avoid global state/access to objects which can lead to fragile code and unexpected behavior. In this case, I consider `TWS` a useful helper class/data type (it is in the root not in `search` for a good reason) grouping certain related attributes together and implementing time window computation logic ; I don't see the problem of using it? On the other hand I think it's not good to 'reimplement' the same logic in Individual (even if it's small ; we already see we messed up with the wait time computation, or at least not computing minimum waiting time). Can we keep it like this?

Reply:
I don't have any particular strong opinion on this; what's the downside of this coupling?  TWS will always be used to compute route(segment) data, so why don't we want to use this?

Except when introducing time-dependent travel times, I am not aware of any commonly used route definitions that cannot be computed by the concatenation scheme in TWS.

Reply:
Actually, one of the reasons I do really want `Solution` to use TWS is time-dependent travel times, since I can then swap out TWS and have consistent time computations in both `Solution` and `LocalSearch` rather than implementing that (much more complicated) logic twice. So @N-Wouda can we keep this?